pages/in/register/index.vue

```diff
@@ -191,6 +191,16 @@ export default {
     },
   },
   async mounted() {
+    const { redirect } = this.$route.query;
+    if (redirect) {
```

Comment:
Can we reuse `tryPostLoginRedirect` ?

Reply:
But in `tryPostLoginRedirect ` we only allow `REDIRECT_WHITE_LIST` to be redirected

Reply:
Got it, let me see see how to reuse

Reply:
Or just implement the white list here

Reply:
[🥅 Validate redirect URL](https://github.com/likecoin/like-co/pull/1742/commits/e34ca0ce03f34f01196e2cb221e6896a239b572d)